src/runtime-rs/crates/hypervisor/src/device/driver/vfio.rs

```diff
@@ -377,24 +408,29 @@ impl VfioDevice {
             .context("get vfio device details failed")?;
 
         // It's safe as BDF really exists.
```

Comment:
I think we should correct the comment as it seems that the BDF not really exists in ccw case, right ? if I am correct, we should add a comment clearly explain the two cases.